### PR TITLE
Ascanrules - SQL rule should not target NoSQL Dbs

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- SQL rule should not target NoSQL Dbs.
```
Comment (kingthorin) [inline addOns/ascanrules/CHANGELOG.md:8]: Should this be done in all the SQLi rules?

Comment (psiinon) [reply addOns/ascanrules/CHANGELOG.md:8]: Not according to https://www.zaproxy.org/techtags/db.mongodb/ 😁

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: addOns/ascanrules/CHANGELOG.md (hunk 1 above)
Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md:8]: Okay

**Review by kingthorin — APPROVED** [review]

**Pull request merged**